tests/e2e/machine_pool_test.go

```diff
@@ -71,5 +69,50 @@ var _ = Describe("TF Test", func() {
 				Expect(respBody.ID()).To(Equal(mp_out.Name))
 			})
 		})
+		Context("Author:amalykhi-High-OCP-64905 @OCP-64905 @amalykhi", func() {
+			It("Edit/delete second machinepool labels", func() {
+				By("Create additional machinepool with labels")
+				replicas := 3
+				machine_type := "r5.xlarge"
+				name := "testmp1"
+				labels1 := map[string]string{"fo1": "bar1", "fo2": "baz2"}
+				labels2 := map[string]string{"fo1": "bar3", "fo3": "baz3"}
+				labels0 := map[string]string{}
+				MachinePoolArgs := &exe.MachinePoolArgs{
+					Token:       token,
+					Cluster:     clusterID,
+					Replicas:    replicas,
+					MachineType: machine_type,
+					Name:        name,
+					Labels:      labels1,
+				}
+
+				err := mpService.Create(MachinePoolArgs)
+				By("Verify the parameters of the created machinepool")
+				mp_response, err := cms.RetrieveClusterMachinePool(conn.RHCSConnection, clusterID, name)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(mp_response.Status()).To(Equal(http.StatusOK))
+				respBody := mp_response.Body()
+				Expect(respBody.Labels()).To(Equal(labels1))
+				By("Edit the labels of the machinepool")
+				MachinePoolArgs.Labels = labels2
+				err = mpService.Create(MachinePoolArgs)
```

Comment:
You are not expecting error to not happen

Reply:
@xueli181114 Do you mean add Expect(err).ToNot(HaveOccurred())?

Reply:
Yes